activator/steplib/activate.go

```diff
@@ -4,57 +4,51 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
 	"slices"
+	"time"
 
 	"github.com/bitrise-io/go-utils/command"
 	"github.com/bitrise-io/go-utils/pathutil"
 	"github.com/bitrise-io/stepman/models"
 	"github.com/bitrise-io/stepman/stepman"
 )
 
-var errStepNotAvailableOfflineMode error = fmt.Errorf("step not available in offline mode")
```

Comment:
Inlined this error handling into `activateStepSource()` because offline mode needs to be reworked for step executables anyway (we only cache the step source now, we'll need to cache the executables themselves)